packages/inter-protocol/src/proposals/utils.js

```diff
@@ -67,7 +68,7 @@ export const reserveThenDeposit = async (
   console.info('confirmed deposit for', debugName);
 };
 
-/** @type {<T>(store: ERef<MapStore>, key: string, make: () => T) => Promise<T>} */
+/** @type {<T>(store: ERef<Map<string, T>>, key: string, make: () => T) => Promise<T>} */
```

Comment:
I'm surprised we have a `Map` based store around

Reply:
Good idea. Done.